src/EventHub/EventHub.Autorest/README.md

```diff
@@ -460,3 +457,64 @@ directive:
 
   - model-cmdlet:
     - KeyVaultProperties
+
+  - where:
+      parameter-name: IdentityType
+    set:
+      breaking-change:
+        old-parameter-type: CaptureIdentityType
+        new-parameter-type: String
+        deprecated-by-version: 7.0.0
```

Comment:
The current Az.EventHub version is 4.2.0 on PSGallery. So the deprecated EventHub version should be 5.0.0 since you don't have preview. Please update the version number accordingly.